hypertrace-view-creator/src/main/resources/configs/raw-trace-view/application.conf

```diff
@@ -18,6 +18,9 @@ pinot.controllerPort = 9000
 pinot.timeColumn = start_time_millis
 pinot.timeUnit = MILLISECONDS
 pinot.dimensionColumns = [tenant_id, duration_millis, start_time_millis, end_time_millis, services, transaction_name, trace_id, num_spans, num_services]
+pinot.rangeIndexColumns = [start_time_millis]
+pinot.bloomFilterColumns = [trace_id]
```

Comment:
As it turns out, bloomFilters don't work on BYTES columns so this list has to be empty. In other config files too.

Reply:
Ok will update